_locales/_untranslated_en.json

```diff
@@ -34,5 +34,17 @@
   },
   "switch_account_desktop":{
     "message": "Switch account"
+  },
+  "select_profile_image_desktop": {
+    "message": "Select your new profile image"
+  },
+  "a11y_profile_image_label":{
```

Comment:
do we really need a a11y string for every button and control? maybe, we could just use the normal "Your profile image" or just "Profile image" string here.

Reply:
every button that will have no text, plan is to use a symbol instead of the text.

Reply:
k, but the having a simple `your_profile_image` would do the job - if can be used as a input field label as well as for a11y. i would use explicit a11y strings only if they are really different from some labels, same as for _desktop, _android etc. suffixes.